app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Buttons -->
+
+        <!-- Primary -->
+
+    <style name="App.Button.Primary" parent="@style/Widget.AppCompat.Button.Colored">
+        <item name="android:layout_width">match_parent</item>
```

Comment:
remove layout_witdh and layout_height from all styles where it hasn't be specified by design